package.json

```diff
@@ -8,7 +8,7 @@
     "okta",
     "oauth2",
     "oidc",
-    "octa-auth-js"
+    "okta-auth-js"
```

Comment:
Ha! Spelling mistakes galore!  Thanks @ryanrishi!

Reply:
You're welcome! This should make it show up when searching NPM and EmberObserver. Thanks for a cool addon!